site/content/en/docs/user-manual/command-reference/util.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 'Utilities'
+linkTitle: 'Utilities'
+description: ''
+weight: 7
+---
+
+### Split video into frames <a id="split-video"></a>
+
+Splits a video into separate frames and saves them in a directory.
+After the splitting, the images can be added into a project using
+the [`import` command](./sources/#source-import) and the `image_dir` format.
+
```

Comment:
I see that `split_video` -> `image_dir` and `video_frames` that two alternative approaches to import video dataset.
Maybe it will be useful to add import argument `--subset` for `image_dir` format to make these approaches fully alternative

Reply:
The both have this parameter. Do you mean, in CLI?

Reply:
Yes, I mean in CLI

Reply:
Done